stubs/pyinstaller/PyInstaller/building/splash.pyi

```diff
@@ -8,24 +8,26 @@ splash_requirements: list[str]
 # Not to be imported during runtime, but is the type reference for spec files which are executed as python code
 class Splash(Target):
     image_file: str
-    full_tk: Incomplete
-    name: Incomplete
-    script_name: Incomplete
-    minify_script: Incomplete
-    max_img_size: Incomplete
-    text_pos: Incomplete
-    text_size: Incomplete
-    text_font: Incomplete
-    text_color: Incomplete
-    text_default: Incomplete
-    always_on_top: Incomplete
-    uses_tkinter: Incomplete
-    script: Incomplete
-    splash_requirements: Incomplete
+    full_tk: bool
+    tcl_lib: str
+    tk_lib: str
+    name: str
+    script_name: StrPath
+    minify_script: bool
+    max_img_size: tuple[int, int]
+    text_pos: tuple[int, int] | None
+    text_size: int
+    text_font: str
+    text_color: str
+    text_default: str
+    always_on_top: bool
+    uses_tkinter: bool
+    script: str
+    splash_requirements: set[str]
     binaries: list[_TOCTuple]
     def __init__(
         self,
-        image_file: StrPath,
+        image_file: str,
```

Comment:
As mentioned in https://github.com/python/typeshed/pull/13568#discussion_r1976504351, `image_file` is only passed to functions that support `PathLike[str]` before being converted to a `str`. This didn't need to change and can be left as:
```suggestion
        image_file: StrPath,
```

Source: https://github.com/pyinstaller/pyinstaller/blob/cef6b7f855494e7283b65f88cb1d7e922d45af66/PyInstaller/building/splash.py#L147-L149